src/modules/WaypointModule.cpp

```diff
@@ -2,6 +2,11 @@
 #include "NodeDB.h"
 #include "PowerFSM.h"
 #include "configuration.h"
+#ifdef HAS_SCREEN
+#include "gps/RTC.h"
+#include "graphics/Screen.h"
+#include "main.h"
+#endif
```

Comment:
It looks like this is causing CI to fail for some targets too, because of the `drawFrame` parameters. I took a peek at the EnvironmentTelemetry module is doing, but it looks like it's just including the OLED library, screen or no screen, which doesn't seem quite right either.